app/views/govuk_publishing_components/components/_step_by_step_nav_related.html.erb

```diff
@@ -4,14 +4,14 @@
   always_display_as_list ||= false
 %>
 <% if links.any? %>
-  <div 
-    class="gem-c-step-nav-related <%= "gem-c-step-nav-related--singular" if links.length == 1 %>" 
+  <div
+    class="gem-c-step-nav-related <%= "gem-c-step-nav-related--singular" if links.length == 1 %>"
     data-module="gem-track-click">
     <h2 class="gem-c-step-nav-related__heading">
       <span class="gem-c-step-nav-related__pretitle"><%= pretitle %></span>
       <% if links.length == 1 && !always_display_as_list %>
           <a href="<%= links[0][:href] %>"
-            class="gem-c-step-nav-related__link"
+            class="gem-c-step-nav-related__link govuk-link"
```

Comment:
If the only thing the `gem-c-step-nav-related__link` class did was extend `govuk-link`, is it even necessary anymore? Could the class on this element just be `govuk-link`?

Reply:
Very probably. I'll check once I've got this working with [alphagov/frontend](https://www.github.com/alphagov/frontend), then I'll optimise.

Reply:
I've removed the `gem-c-step-nav-related__link` from this component. This was quite tied into the tests, but aside from that I couldn't find a reason to keep the class there.